src/main/java/core/basesyntax/ConsoleHandler.java

```diff
@@ -0,0 +1,165 @@
+            String[] split = input.split(" ");
+            switch (split[0]) {
+                case "create":
+                    create(split[1], split[2]);
+                    break;
+                case "read":
+                    read(split[1], split[2]);
+                    break;
+                case "info":
+                    info(split[1], split[2]);
+                    break;
+                case "help":
+                    if (split.length >= 2) {
+                        help(split[1]);
+                    } else {
+                        help();
+                    }
+                    break;
+                case "exit":
+                    System.exit(0);
+                    break;
+                default:
+                    saveText(input);
+            }
+        }
+    }
+
+    private void info(String path, String fileName) throws IOException {
+        Path file = Path.of(path + File.separator + fileName);
+        System.out.println(Files.readAllBytes(file).length + " symbols");
+        System.out.println(Files.readAllLines(file).size() + " lines");
+        System.out.println(countWords(Files.readAllLines(file)) + " words");
+        Calendar calendar = Calendar.getInstance();
+        BasicFileAttributes attributes = Files.readAttributes(file, BasicFileAttributes.class);
+        calendar.setTimeInMillis(attributes.lastModifiedTime().toMillis());
+
+        System.out.println(calendar.getTime() + " last modified");
+        System.out.println("Size: " + attributes.size() + " bytes");
+    }
+
+    private long countWords(List<String> lines) {
+        //this is a hw for lesson 5, so students are not obliged to use Stream API
+        return lines.stream()
+                .map(line -> Stream.of(line.split(" ")))
+                .flatMap(o -> o)
+                .filter(str -> str.matches("^[a-zA-Z]+"))
+                .count();
+    }
+
+    private void help() {
+        for (String info : commandInfo.values()) {
+            System.out.println(info);
+        }
+    }
+
+    private void help(String commandName) {
+        System.out.println(commandInfo.get(commandName));
+    }
+
+    private void saveText(String input) throws IOException {
+        if (input.isEmpty()) {
+            return;
+        }
+        System.out.println("Save text? Y/N");
+        if (askYesNo()) {
+            String[] saveFilePath = getSaveFilePath();
+            create(saveFilePath[0], saveFilePath[1]);
+            writeToFile(saveFilePath[0], saveFilePath[1], input);
+        }
+    }
+
+    private boolean askYesNo() {
+        Scanner scanner = new Scanner(System.in);
+        String answer = scanner.next();
+        switch (answer) {
+            case "Y":
+                return true;
+            case "N":
+                return false;
+            default:
+                System.out.println("Unknown answer. Please, write Y or N");
+                askYesNo();
```

Comment:
here you are ignoring user input. and even if users enters Y method will return false.